source/extensions/filters/http/well_known_names.h

```diff
@@ -80,6 +80,8 @@ class HttpFilterNameValues {
   const std::string AwsLambda = "envoy.filters.http.aws_lambda";
   // OAuth filter
   const std::string OAuth = "envoy.filters.http.oauth";
+  // Local Rate limit filter
+  const std::string LocalRateLimit = "envoy.filters.http.local_ratelimit";
```

Comment:
Trying to get rid of this file. Can you just inline this string where it's used and not depend on this WKN file?